buffer/src/main/java/io/netty5/buffer/adapt/AdaptivePoolingAllocator.java

```diff
@@ -326,8 +393,10 @@ private void rotateHistograms() {
             int percentileSize = 1 << sizeBucket + HISTO_MIN_BUCKET_SHIFT;
             int prefChunkSize = Math.max(percentileSize * BUFS_PER_CHUNK, MIN_CHUNK_SIZE);
             localPrefChunkSize = prefChunkSize;
-            for (Magazine mag : parent.magazines) {
-                prefChunkSize = Math.max(prefChunkSize, mag.localPrefChunkSize);
+            if (ownerEventExecutor != null) {
```

Comment:
```suggestion
            if (ownerEventExecutor == null) {
```

to follow the logic of the 4.1 `shareable` field, right?